src/js/Component/DataList.js

```diff
@@ -9,7 +11,12 @@ Wui.DataList = function(args){
         focusOnSelect:  true,
         interactive:    true,
         multiSelect:    false,
-        selected:       []
+        selected:       [],
```

Comment:
Comments should go on the line above (see [Style Guide rule 5.1](http://www.wui-js.com/js-style-guide.html#5_1))